.semaphore/semaphore.yml

```diff
@@ -34,9 +34,6 @@ blocks:
           value: "test"
         - name: TB_RSPEC_OPTIONS
           value: "--profile --format RspecJunitFormatter --out junit.xml"
-      secrets:
```

Comment:
Could please explain the reason why you deleted these 'secrets' here?

Reply:
@prabhanshuguptagit and I were pairing to resolve CI issues blocking this merge, and he noticed that these secrets were not used in this job, so we removed it. Forgot to update the description to mention this. 

In an ideal world, this would go in a separate PR, but this also gave us an excuse to retrigger the job as well (:

Reply:
Learnt that this broke something. I'll raise a new PR without this change. Sorry folks.